src/CMakeLists.txt

```diff
@@ -283,7 +284,6 @@ ncnn_add_layer(Quantize)
 ncnn_add_layer(Dequantize)
 ncnn_add_layer(Yolov3DetectionOutput)
 ncnn_add_layer(PSROIPooling)
-ncnn_add_layer(ROIAlign OFF)
```

Comment:
Do not change the order
the generated layer index enum order breaks compatibility